src/Wallet/BaseWallet.ts

```
@@ -0,0 +1,218 @@
+    }
+
+    signAminoSupport(){
+        return !!this.signer?.signAmino
+    }
+
+    getOptions(): any {
+        return this.options;
+    }
+
+    setOptions(options: any): any {
+        return this.options = options;
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)